src/routes/publish.js

```diff
@@ -118,23 +125,7 @@ export function publishRoutes() {
     // Koa which will expose it appropriately to the caller.
     await Promise.all([pipelinePromise, jsonCheckerPromise]);
 
-    const storage = gcsStorageCreate(config);
-    const hash = hasherTransform.sha1();
-
-    const googleStorageStream = storage.getWriteStreamForFile(hash);
-
-    // We can't use Stream.finished here because of a problem with Google's
-    // library. For more information, you can see
-    // https://github.com/googleapis/nodejs-storage/issues/937
-    await new Promise((resolve, reject) => {
-      const fullContent = concatener.transferContents();
-      googleStorageStream.once('error', reject);
-      googleStorageStream.once('finish', resolve);
-      googleStorageStream.end(fullContent);
-    });
```

Comment:
I started looking at implementing this when I found memory and leaking problems with this. I think that calling `end` with the `fullContent` isn't good, because then we can't make it stop if for example the original request is aborted.

I tried to fix just this in #90, but then thought it was just easier to do the Right Thing.